.github/workflows/flutter_ios.yaml

```diff
@@ -0,0 +1,68 @@
+name: Flutter iOS
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        description: The working directory relative to the repo root
+        required: true
+        type: string
+      package-name:
+        description: The name of the app-facing (or federated) iOS package being tested
+        required: true
+        type: string
+
+# Cancels in-progress job when there is another push to same ref.
```

Comment:
callout: We do this in integration tests but I noticed we don't do it many unit test flows. Getting the result of a previous commit is nice to know when iterating on something, but IMO macos targets can get scarce so better to add this to conserve. I'm open to removing this here or if ppl think we should add more widely to unit test jobs.